uptane-whitepaper.md

```diff
@@ -1 +1,106 @@
+
+**Awareness of evolving regulations and standards**--As government and industry begins to awaken to the need for improved automotive security, Uptane is seeking to stay one step ahead of emerging regulations and standards governing over the air updates and other aspects of cybersecurity.  
+
+## **Starting conversations about the challenges ahead**
+As vehicles edge ever closer to the designs envisioned in novels and movies in the mid-20th century, there will be new questions to address. Currently, the Uptane group is opening conversation on two such issues. Look for new whitepapers at the end of 2021 and in early 2022 to deal with the following issues.
+
+*Allowing access to ECUs for emergency updates from federal/state/local governments*
+As SOTA delivery of updates becomes more sophisticated, government agencies and regulatory bodies, such as the U.S. Department of Transportation or its state or local equivalents, the Department of Homeland Security, or the Federal Emergency Management Agency,  may require automakers to grant them full control over a vehicle in emergency situations. These infrequent but important OTA broadcasts could be due to emergency or disaster routing, scheduled road work, or traffic conditions due to closed exits, failed stoplights or other conditions.  Accommodating this access will require some re-thinking of how Uptane is configured, particularly in how to prioritize delegations, and perhaps to accommodate a dual director set-up.
+
+*Security issues related to the use of aftermarket materials*
+Aftermarket companies refurbish and reuse equipment following end-of-life support from OEMs. This means introducing ECUs to a vehicle that the OEM has no control over. In addition, because aftermarket suppliers may not have access to the original design, they often reverse engineer the parts to figure out how it works. 
+Such an approach perhaps keeps these suppliers from being able to glean all relevant design information about the ECU.
+
+The white paper will address a number of issues, including the answers to the following questions:
```

Comment:
Is this for the aftermarket white paper?

Reply:
I need to take a closer look at what happened there. I think I wanted to allude to new direction/future work, but maybe I accidentally left some out.

Reply:
I just checked. Yes the text above comes at the end where we preview upcoming issues we'll be addressing